liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/DevTest.java

```diff
@@ -136,6 +136,7 @@ public void resourceFileChangeTest() throws Exception {
    @Test
    public void testDirectoryTest() throws Exception {
       tagLog("##testDirectoryTest start");
+      Thread.sleep(500); // this test often fails, wait for dev mode
```

Comment:
Is this test often failing because it starts before dev mode is actually "ready"?

Reply:
Yes, I believe so. Issue 1554 is tracking this.